README.md

```diff
@@ -38,15 +59,16 @@ The ROS package can be installed and invoked by the following commands.
 
 ### Installation
 
-```
+```sh
 git clone https://github.com/jk-ethz/franka_lock_unlock.git
 colcon build --packages-up-to franka_lock_unlock
 ```
 
 ### Simple Usage
 
 ```sh
-ros2 run franka_lock_unlock franka_lock_unlock.py <PARAMS>
+ros2 run franka_lock_unlock lock_unlock.py <PARAMS>
```

Comment:
Not sure if I am a fan of this. I'd rather keep this name as the file is named, i. e. `franka_lock_unlock.py` and `franka_shutdown.py`.
Then people will get immediately which file the run corresponds to, and don't need to check the `setup.py` for it.

Reply:
Done in d9ebd552cf8e924fe3aac08fad5c459804607851.